pkg/roachpb/api.proto

```diff
@@ -1470,7 +1471,7 @@ message ExternalStorage {
     // Path is the filename being read/written to via the FileTableSystem.
     string path = 3;
   }
-  LocalFilePath LocalFile = 2 [(gogoproto.nullable) = false];
+  LocalFileConfig local_file_config = 2 [(gogoproto.nullable) = false];
```

Comment:
I suppose that's fine -- should be backward compatible..

Reply:
Yup, just a rename and pulling it out of a message since we're using it in more than just ExternalStorage now.